controllers/articles-controller.js

```diff
@@ -1,4 +1,7 @@
-const { selectArticles } = require("../models/articles-model");
+const {
+  selectArticles,
+  selectAllArticles,
+} = require("../models/articles-model");
 
 exports.getArticles = (req, res, next) => {
```

Comment:
if you're getting a single article, the function should reflect that it is a single result not plural.

Reply:
<img width="164" alt="Screenshot 2024-08-26 at 13 32 59" src="https://github.com/user-attachments/assets/da123819-5f13-4ff3-bb59-9d4046a9f14f">